code/datums/components/chem_booster.dm

```diff
@@ -187,9 +187,11 @@
 	wearer.heal_limb_damage(6*boost_amount*brute_heal_amp, 6*boost_amount*burn_heal_amp)
 	if(connected_weapon && world.time - processing_start < 20 SECONDS)
 		wearer.adjustStaminaLoss(-7*stamina_regen_amp*((20 - (world.time - processing_start)/10)/20)) //stamina gain scales inversely with passed time, up to 20 seconds
-	if(world.time - processing_start > 12 SECONDS && world.time - processing_start < 15 SECONDS)
-		wearer.overlay_fullscreen("degeneration", /obj/screen/fullscreen/infection, 1)
-		to_chat(wearer, span_highdanger("WARNING: You have [(200 - (world.time - processing_start))/10] seconds before necrotic tissue forms on your limbs."))
+	if(world.time - processing_start > 10 SECONDS && world.time - processing_start < 20 SECONDS)
+		to_chat(wearer, span_bold("WARNING: You have [(200 - (world.time - processing_start))/10] seconds before necrotic tissue forms on your limbs."))
+		if(world.time - processing_start > 15 SECONDS && world.time - processing_start < 20 SECONDS)
```

Comment:
You are checking the same thing twice. And also you can cache world.time - processing start in a var